src/App.js

```diff
@@ -82,7 +87,19 @@ if (__DEV__) {
 } else {
   // eslint-disable-next-line no-inner-declarations
   async function initSentry() {
```

Comment:
I guess the name `initSentry` is a bit outdated now?

Reply:
yea